packages/beacon-node/src/util/index.ts

```diff
@@ -0,0 +1,2 @@
+//  Only export the functions which are useful for external packages and difficult to reproduce or mock
+export * from "./kzg.js";
```

Comment:
Where do we need this?

Reply:
To create sim tests for 4844, we need trusted setup. To create and sign blobs.